openlibrary/plugins/openlibrary/home.py

```diff
@@ -86,20 +96,39 @@ def get_ia_carousel_books(query=None, subject=None, work_id=None, sorts=None,
         query = CAROUSELS_PRESETS[query]
 
     limit = limit or lending.DEFAULT_IA_RESULTS
-    books = lending.get_available(limit=limit, subject=subject, work_id=work_id,
-                                  _type=_type, sorts=sorts, query=query)
+    books = lending.get_available(
+        limit=limit,
+        subject=subject,
+        work_id=work_id,
+        _type=_type,
+        sorts=sorts,
+        query=query,
+    )
     formatted_books = [format_book_data(book) for book in books if book != 'error']
     return formatted_books
 
+
 def get_featured_subjects():
     # web.ctx must be initialized as it won't be available to the background thread.
     if 'env' not in web.ctx:
         delegate.fakeload()
 
     FEATURED_SUBJECTS = [
-        'art', 'science_fiction', 'fantasy', 'biographies', 'recipes',
-        'romance', 'textbooks', 'children', 'history', 'medicine', 'religion',
-        'mystery_and_detective_stories', 'plays', 'music', 'science'
+        'art',
+        'science_fiction',
+        'fantasy',
+        'biographies',
+        'recipes',
+        'romance',
+        'textbooks',
+        'children',
+        'history',
+        'medicine',
+        'religion',
+        'mystery_and_detective_stories',
+        'plays',
+        'music',
+        'science',
```

Comment:
No unrelated formatting noise please.

Reply:
Ok, sure, will revert this back to the original format.